RIS.js

```diff
@@ -1266,6 +1268,21 @@ function processTag(item, tagValue, risEntry) {
 				zField = ['extra'];
 			}
 		break;
+		case "H1":
+			//multiple occurences are not saved
+			if(item.libraryCatalog) {
+				item.multipleLibraries = value;
+			}
+		break;
+		case "H2":
+			//multiple occurences are not saved
+			if(item.callNumber) {
+				zField = ['notes'];
+				if(item.multipleLibraries) {
+					value = item.multipleLibraries+": "+value;
+				}
+			}
+		break;
```

Comment:
This means, that unsupported RIS fields are ignored by default. What value should I choose to run the tests?